tests/test_repository.py

```diff
@@ -144,6 +145,11 @@ def test_sle_bci_forbidden_packages(container_per_test):
         "nfs-kernel-server",
         "texlive-l3kernel.noarch",
         "purge-kernels-service.noarch",
+        f"kernel-azure-devel.{LOCALHOST.system_info.arch}",
+        "kernel-devel-azure.noarch",
+        "kernel-macros.noarch",
+        "kernel-source-azure.noarch",
+        f"kernel-syms-azure.{LOCALHOST.system_info.arch}",
```

Comment:
Shouldn't this also have a `.noarch` package or is this special?

Reply:
Apparently it's special:
```ShellSession
❯ podman run --rm -it registry.suse.com/bci/bci-base:15.5 zypper info kernel-syms-azure|grep Arch
Arch           : x86_64
```